pages/wizard/regions/navigation_buttons.py

```diff
@@ -0,0 +1,138 @@
+import re
+from selenium.webdriver.common.by import By
+from pages.dashboard import DashboardPage
+
+class NavigationButtons():
+    """
+    Implements the controls for the navigation buttons on the
+    wizards different pages.   Most pages have:
+
+        - back
+        - cancel
+        - next
+
+    Clicking cancel will open a modal frame, with the buttons:
+
+       - Exit and Delete
+       - Exit and Save
+       - Continue Working
+
+    All the locators are contained in here with functions to access
+    them.
+    """
+
+    def __init__(self, base_url, selenium):
+        self.base_url = base_url
+        self.selenium = selenium
+
+    # locators:
+    _cancel_loc = (
+        By.XPATH,
+        '//button[contains(@class,"btn") and contains(., "Cancel")]'
+    )
+    _back_loc = (
+        By.XPATH,
+        '//a[contains(@class, "btn") and contains(., "Back")]'
+    )
+    _next_loc = (
+        By.XPATH,
+        '//button[contains(@class,"btn") and contains(., "Next")]'
+    )
+
+    # These locators for the cancel model frame.   Note since the code
+    # is always there we don't need to select the dialogue as its just
+    # a frame in the existing page that sometimes is visible.
+    _exit_and_delete_loc = (
+        By.XPATH,
+        '//button[contains(.,"Exit and Delete")]'
+    )
+    _exit_and_save_loc = (
+        By.XPATH,
+        '//button[contains(.,"Exit and Save")]'
+    )
+    _continue_working_loc = (
+        By.XPATH,
+        '//button[contains(.,"Continue Working")]'
+    )
+
+    # properties
+    @property
+    def cancelBtn(self):
```

Comment:
Right...this is the sucky part of this...I moved them to a different file so the previous fix just kind of goes way.  On it.